atividade03/src/main/java/br/edu/uepb/eda/atividade03/FilaComPilhas.java

```diff
@@ -0,0 +1,57 @@
+package br.edu.uepb.eda.atividade03;
+
+public class FilaComPilhas implements Fila_IF {
+
+    private PilhaComLista p1;
+    private PilhaComLista p2;
+
+    public FilaComPilhas() {
+        this.p1 = new PilhaComLista();
+        this.p2 = new PilhaComLista();
+    }
+
+   @Override
+	public void enqueue(Integer element) throws Exception {
+		if (isFull()) {
+			throw new Exception("Fila está cheia.");
+		} else {
+			while(!p1.isEmpty()) {
+				p2.push(p1.pop());
+			}
+			p2.push(element);
+			while(!p2.isEmpty()) {
+				p1.push(p2.pop());
+			}
+		}
+	}
+
+    @Override
+    public Integer dequeue() throws Exception {
+        if (isEmpty()) {
+            throw new Exception("Nao da para remover pois esta vazia.");
+        }else {
+			int dequeued = p1.pop();
```

Comment:
```suggestion
			ireturn p1.pop();
```

Poderia fazer o retorno diretamente, assim seria menos um váriavel alocada em memória.